### DISCUSSION TITLE
Ultra fast image generation with v1.0.0 beta 28 release (820 milliseconds/ 0.82seconds)

### DISCUSSION THREAD

**Discussion by rupeshs** [rupeshs/fastsdcpu#164] [Announcements]
![fastsdccpu-sdxs0 9](https://github.com/rupeshs/fastsdcpu/assets/3255994/7a5aa531-87c8-410c-bf62-6eddca82d953)

Release: https://github.com/rupeshs/fastsdcpu/releases/tag/v1.0.0-beta.28

**Comment by patientx** [comment]
> ![fastsdccpu-sdxs0 9](https://private-user-images.githubusercontent.com/3255994/317313396-7a5aa531-87c8-410c-bf62-6eddca82d953.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE1NzMwNjksIm5iZiI6MTcxMTU3Mjc2OSwicGF0aCI6Ii8zMjU1OTk0LzMxNzMxMzM5Ni03YTVhYTUzMS04N2M4LTQxMGMtYmY2Mi02ZWRkY2E4MmQ5NTMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMyNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMjdUMjA1MjQ5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NWQ3NjQ1Y2Q3MDFlZjc2ZjE2NWZmNmNmYjcwZDBmMjUwN2I0ODcxZjQ5YWRiMmY3ODVjNDZjODhhYjQ2MWQ4OCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.gw2FUyVza-S_GjEBxNoGhBnnhJRfwfWsh7h7JTZ9juE)
> 
> Release: https://github.com/rupeshs/fastsdcpu/releases/tag/v1.0.0-beta.28

fyi, in android method with termux latest available mediapipe version is 0.10.9, latest requirements.txt says we need 10.11 but since that is not available I  changed it and tried 0.10.9, and so far at least with basic turbo , lcm and sdxs generation works, I don't know what requires mediapipe 0.10.11 (maybe control.net or upscaling? , never tried those on the phone) but at least basic generation still works. Just wanted to let you know. 

Also for comparison , it takes almost half the time compared to turbo model for 1 step generation but the result is a bit ... meh ... (not that bad but turbo is just a little better at least for this 0.9 version, they say there will a 1.0 we'll wait and see)

**Reply by rupeshs** [reply]
Yes, ControlNet preprocessor needs mediapipe. They will probably release SDXS-512-1.0 ,see this discussion https://github.com/IDKiro/sdxs/issues/4

**Comment by rupeshs** [comment]
@patientx I just reverted to mediapipe to version 0.10.9.

**Reply by patientx** [reply]
thanks, good to know it still works with that.
